packages/ia-topnav/src/nav-search.js

```diff
@@ -38,6 +41,14 @@ class NavSearch extends TrackedElement {
     return true;
   }
 
+  initSearchBetaOptIn() {
+    try {
+      this.inSearchBeta = JSON.parse(window.localStorage?.getItem('SearchBeta-opt-in'));
+    } catch (err) {
```

Comment:
whoa localStorage instead of cookies, eh?  neat. :)

Reply:
Yeah, I figure since the opt in/out preference is entirely a client-side affair there's little reason for it to be sent as a cookie with every request! (The AB test system in petabox has no knowledge of opt in/out, only beta link visibility)